Gemfile

```diff
@@ -24,7 +24,7 @@ end
 
 gem 'dhasher'
 gem 'discard', :git => 'https://github.com/jhawthorn/discard', :branch => 'master'
-gem 'insights-api-common', '~> 4.0'
+gem 'insights-api-common', '~> 4.1.4'
```

Comment:
Do we really need this change. With `~> 4.0`, it automatically upgrades to 4.1.4

Reply:
@bzwei it depends if you want to force people to update. If you don't, I can closed it

Reply:
@pkomanek Yes we can close it for now. Thanks.